apps/condo/bin/delete-meter-readings-by-property-ids.js

```diff
@@ -0,0 +1,87 @@
+const path = require('path')
```

Comment:
you also need to add ./bin/README.md example

Reply:
README.md example added